kernel/tests/integration-test.rs

```diff
@@ -2282,3 +2282,29 @@ fn s06_leveldb_benchmark() {
         check_for_successful_exit(&cmdline, qemu_run(), output);
     }
 }
+
+/// Tests that basic pmem allocation support is functional.
+#[test]
+fn s06_pmem_alloc() {
+    let machine = Machine::determine();
```

Comment:
probably want to make sure that the test can persist data durably too eventually?

Reply:
Added a TODO statement.